src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -122,7 +122,12 @@ export const NavBarSchedule: React.FC<NavBarScheduleProps> = ({
             setSelectedDayIndex(dayIndex);
           }}
         >
-          {formatDayLabel(day)}
+          <button
+            aria-label={getWeekday(day)}
+            className="NavBarSchedule__weekday-button"
+          >
+            {formatDayLabel(day)}
+          </button>
```

Comment:
Do you really need this button? The `onClick` event is passed to the parent node `li`. Is this going to have any impact at all?

Reply:
@denisdimitrov the button is for accessibility purposes

Reply:
yes. The focus works for buttons. Not for li's
![image](https://user-images.githubusercontent.com/84011566/125602805-e5bad6a1-bc5c-4419-9f22-274885719c85.png)

Reply:
But the onClick’s should be moved to the buttons, and not be on the li’s